examples/cms-buttercms/next.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  reactStrictMode: true,
+  async rewrites() {
+    return [
+      {
+        source: '/',
+        destination: '/landing-page/landing-page-with-components',
+      },
+    ]
+  },
+  redirects() {
+    const sourcesRequiringAuthToken = [
+      '/',
+      '/landing-page/:slug*',
+      '/blog/:path*',
+    ]
+
+    return process.env.NEXT_PUBLIC_BUTTER_CMS_API_KEY
+      ? [
+          {
+            source: '/missing-token',
+            destination: '/',
+            permanent: false,
+          },
+        ]
+      : sourcesRequiringAuthToken.map((source) => ({
+          source: source,
+          destination: '/missing-token',
+          permanent: false,
+        }))
+  },
+  images: {
+    domains: ['cdn.buttercms.com'],
+    dangerouslyAllowSVG: true,
```

Comment:
Do we need this?

Reply:
I believe we need the domains specification as the images for the landing page come from the CDN, but I've deleted the dangerously allow SVG key